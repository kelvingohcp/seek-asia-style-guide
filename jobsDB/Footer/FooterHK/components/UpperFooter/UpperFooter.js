import React from 'react';
import PropTypes from 'prop-types';

import styles from './UpperFooter.less';
import PartnerSites from '../PartnerSites/PartnerSites';
import { Columns, ListItem, Section, Text, FacebookIcon, TwitterIcon, AppleIcon, AndroidIcon, Hidden } from 'seek-asia-style-guide/react';

const UpperFooter = ({ messages, wordpressLink, cfsLink, externalLink, hasCompanyProfile }) => {
  return (
    <Columns>
      <div className={styles.columnGroup}>
        <Section className={styles.firstCategory}>
          <Text waving semiStrong className={styles.title}>
            {messages['FooterHK.titleAboutJobsDB']}
          </Text>
          <Hidden mobile={true}>{ wordpressLink(
            <ListItem value={messages['FooterHK.aboutJobsDB']} disableBackground compact noShadow descriptionProps={{ semiStrong: true, whistling: true }} />,
            messages['FooterHK.aboutJobsDBLink']
          )}</Hidden>
          <Hidden mobile={true}>{cfsLink(
            <ListItem value={messages['FooterHK.faq']} disableBackground compact noShadow descriptionProps={{ semiStrong: true, whistling: true }} />,
            messages['FooterHK.faqLink']
          )}</Hidden>
          {cfsLink(
            <ListItem value={messages['FooterHK.career']} disableBackground compact noShadow descriptionProps={{ semiStrong: true, whistling: true }} />,
            messages['FooterHK.careerLink']
          )}
          <Hidden desktop={false} tablet={false} mobile={true}>{wordpressLink(
            <ListItem value={messages['FooterHK.contactUs']} disableBackground compact noShadow descriptionProps={{ semiStrong: true, whistling: true }} />,
            messages['FooterHK.contactUsLink']
          )}</Hidden>
          <Hidden desktop={true} tablet={true} mobile={false}>{externalLink(
            <ListItem value={messages['FooterHK.contactUs']} disableBackground compact noShadow descriptionProps={{ semiStrong: true, whistling: true }} />,
            messages['FooterHK.contactUsLinkMsite']
          )}</Hidden>
          <PartnerSites messages={messages} externalLink={externalLink} />
        </Section>
        <Section className={styles.category}>
          <Text waving semiStrong className={styles.title}>
            {messages['FooterHK.titleJobSeeker']}
          </Text>
          {cfsLink(
            <ListItem value={messages['FooterHK.browseJobs']} disableBackground compact noShadow descriptionProps={{ semiStrong: true, whistling: true }} />,
            messages['FooterHK.browseJobsLink']
          )}
          {<Hidden mobile={true}>{cfsLink(
            <ListItem value={messages['FooterHK.resumes']} disableBackground compact noShadow descriptionProps={{ semiStrong: true, whistling: true }} />,
            messages['FooterHK.resumesLink']
          )}</Hidden>}
          <Hidden mobile={true}>{cfsLink(
            <ListItem value={messages['FooterHK.jobAlerts']} disableBackground compact noShadow descriptionProps={{ semiStrong: true, whistling: true }} />,
            messages['FooterHK.jobAlertsLink']
          )}</Hidden>
          {cfsLink(
            <ListItem value={messages['FooterHK.myJobsDBTitle']} disableBackground compact noShadow descriptionProps={{ semiStrong: true, whistling: true }} />,
            messages['FooterHK.myJobsDBUrl']
          )}
          {cfsLink(
            <ListItem value={messages['FooterHK.careerInsightsTitle']} disableBackground compact noShadow descriptionProps={{ semiStrong: true, whistling: true }} />,
            messages['FooterHK.careerInsightsUrl']
          )}
        </Section>
      </div>
      <div className={styles.columnGroup}>
        <Section className={styles.category}>
          <Text waving semiStrong className={styles.title}>
            {messages['FooterHK.titleEmployer']}
          </Text>
          {wordpressLink(
            <ListItem value={messages['FooterHK.postJob']} disableBackground compact noShadow descriptionProps={{ semiStrong: true, whistling: true }} />,
            messages['FooterHK.postJobLink']
          )}
          {wordpressLink(
            <ListItem value={messages['FooterHK.searchCandidates']} disableBackground compact noShadow descriptionProps={{ semiStrong: true, whistling: true }} />,
            messages['FooterHK.searchCandidatesLink']
          )}
          {wordpressLink(
            <ListItem value={messages['FooterHK.advertise']} disableBackground compact noShadow descriptionProps={{ semiStrong: true, whistling: true }} />,
            messages['FooterHK.advertiseLink']
          )}
          {hasCompanyProfile && cfsLink(
            <ListItem value={messages['FooterHK.bestCompanies']} disableBackground compact noShadow descriptionProps={{ semiStrong: true, whistling: true }} />,
            messages['FooterHK.bestCompaniesLink']
          )}
        </Section>
        <Section className={styles.category}>
          <Text waving semiStrong className={styles.title}>
            {messages['FooterHK.titleConnect']}
          </Text>
          {externalLink(
            <ListItem
              value={messages['FooterHK.facebook']}
              disableBackground
              icon={<FacebookIcon className={styles.content} square />}
              compact
              noShadow
              descriptionProps={{ semiStrong: true, whistling: true }}
            />,
            messages['FooterHK.facebookLink']
          )}
          {externalLink(
            <ListItem
              value={messages['FooterHK.twitter']}
              disableBackground
              icon={<TwitterIcon className={styles.content} generic />}
              compact
              noShadow
              descriptionProps={{ semiStrong: true, whistling: true }}
            />,
            messages['FooterHK.twitterLink']
          )}
          {externalLink(
            <ListItem
              value={messages['FooterHK.appStore']}
              disableBackground
              icon={<AppleIcon className={styles.content} />}
              compact
              noShadow
              descriptionProps={{ semiStrong: true, whistling: true }}
            />,
            messages['FooterHK.appStoreLink']
          )}
          {externalLink(
            <ListItem
              value={messages['FooterHK.googlePlay']}
              disableBackground
              icon={<AndroidIcon className={styles.content} />}
              compact
              noShadow
              descriptionProps={{ semiStrong: true, whistling: true }}
            />,
            messages['FooterHK.googlePlayLink']
          )}
        </Section>
      </div>
    </Columns>
  );
};

UpperFooter.propTypes = {
  messages: PropTypes.object.isRequired,
  externalLink: PropTypes.func.isRequired,
  cfsLink: PropTypes.func.isRequired,
  wordpressLink: PropTypes.func.isRequired,
  hasCompanyProfile: PropTypes.bool
};

UpperFooter.defaultProps = {
  hasCompanyProfile: false
};

export default UpperFooter;
