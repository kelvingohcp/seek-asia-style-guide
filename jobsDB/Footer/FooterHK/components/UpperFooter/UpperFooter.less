@import (reference) "~seek-asia-style-guide/theme";
@wrapper-low-vertical-padding: @grid-base * 10;
@title-padding-bottom: @grid-base * 3;

.columnGroup {
  display: flex;
  @media @mobile {
    justify-content: space-evenly;
  }
}

.category {
  padding: 0;
  margin: 0 auto;
  flex: 1;
  @media @mobile {
    margin: inherit;
    padding-bottom: @wrapper-low-vertical-padding;
  }
}

.firstCategory:extend(.category) {
  margin-left: 0;
}

.title {
  color: @dbBrand;
  padding-bottom: @grid-base * 4;
  padding-left: @grid-base * 4;
  padding-right: @grid-base * 4;
}

.content {
  color: @saGrey2;
}
