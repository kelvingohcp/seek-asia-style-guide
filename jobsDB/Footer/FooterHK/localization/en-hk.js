export default {
  'FooterHK.cmsLink': '/en-hk',
  'FooterHK.cfsLink': '/hk/en',
  'FooterHK.titleAboutJobsDB': 'About JobsDB',
  'FooterHK.titleJobSeeker': 'Job Seekers',
  'FooterHK.titleEmployer': 'Employers',
  'FooterHK.titleConnect': 'Connect',
  'FooterHK.copyright': 'Â© jobsDB. All Rights Reserved',
  'FooterHK.siteMapLink': '/sitemap',
  'FooterHK.siteMapLinkText': 'Site Map',
  'FooterHK.termsLink': '/pages/terms/terms-conditions',
  'FooterHK.termsLinkText': 'Terms & Conditions',
  'FooterHK.privacyLink': '/pages/terms/privacy-policy',
  'FooterHK.privacyLinkText': 'Privacy Statement',
  'FooterHK.aboutJobsDB': 'About jobsDB',
  'FooterHK.aboutJobsDBLink': '/pages/aboutus',
  'FooterHK.faq': 'FAQ',
  'FooterHK.faqLink': '/StaticContentFullContainer/AboutUS/faq/index.htm',
  'FooterHK.career': 'Career @ jobsDB',
  'FooterHK.careerLink': '/Search/FindJobs?AccNums=1&JSRV=1&page=1',
  'FooterHK.contactUs': 'Contact us',
  'FooterHK.contactUsLink': '/pages/regional-coverage',
  'FooterHK.partners': 'International partners',
  'FooterHK.browseJobs': 'Browse jobs',
  'FooterHK.browseJobsLink': '/browse',
  'FooterHK.resumes': 'Post resumes',
  'FooterHK.resumesLink': '/Resumes/ResumeIndex',
  'FooterHK.jobAlerts': 'Job Alerts',
  'FooterHK.jobAlertsLink': '/JobAlert/JobAlertCreation',
  'FooterHK.careerInsights': 'Career Insights',
  'FooterHK.postJob': 'Post a job',
  'FooterHK.postJobLink': '/cms/employer/product-services/job-ads/',
  'FooterHK.searchCandidates': 'Search candidates',
  'FooterHK.searchCandidatesLink': '/cms/employer/product-services/talent-search/',
  'FooterHK.advertise': 'Advertise with us',
  'FooterHK.advertiseLink': '/cms/employer/product-and-services/',
  'FooterHK.myJobsDBTitle': 'My jobsDB',
  'FooterHK.myJobsDBUrl': '/MyJobsDB',
  'FooterHK.careerInsightsTitle': 'Career Insights',
  'FooterHK.careerInsightsUrl': '/career-insights',
  'FooterHK.bestCompanies': 'Best Companies',
  'FooterHK.bestCompaniesLink': '/companies',
  'FooterHK.facebook': 'Facebook',
  'FooterHK.facebookLink': 'https://www.facebook.com/JobsDB.HongKong',
  'FooterHK.twitter': 'Twitter',
  'FooterHK.twitterLink': 'https://twitter.com/HKJobs',
  'FooterHK.appStore': 'jobsDB @ App Store',
  'FooterHK.appStoreLink': 'https://itunes.apple.com/hk/app/jobsdb-job-search/id414607432',
  'FooterHK.googlePlay': 'jobsDB @ Google Play',
  'FooterHK.googlePlayLink': 'https://play.google.com/store/apps/details?id=com.jobsdb',
  'FooterHK.BdjobsLink': 'https://www.bdjobs.com/',
  'FooterHK.BrighterMondayLink': 'https://www.brightermonday.com/',
  'FooterHK.CathoLink': 'https://www.catho.com.br/',
  'FooterHK.JobbermanLink': 'https://www.jobberman.com/',
  'FooterHK.JobStreetLink': 'https://www.jobstreet.com/',
  'FooterHK.JoraLink': 'https://hk.jora.com/',
  'FooterHK.OCCLink': 'https://www.occ.com.mx/',
  'FooterHK.SeamanJobsiteLink': 'https://www.seamanjobsite.com/',
  'FooterHK.SEEKLink': 'https://www.seek.com.au/',
  'FooterHK.WorkAbroadLink': 'https://www.workabroad.ph/',
  'FooterHK.WorkanaLink': 'https://www.workana.com/',
  'FooterHK.ZhaopinLink': 'https://www.zhaopin.com/'
};
