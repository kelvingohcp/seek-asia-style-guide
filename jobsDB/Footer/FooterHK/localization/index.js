import enHkLocalisation from './en-hk';

const localization = {
  'en-hk': enHkLocalisation
};

export const getLocalization = ({ language, country }) => {
  return localization[`${language.toLowerCase()}-${country.toLowerCase()}`] ?
    localization[`${language.toLowerCase()}-${country.toLowerCase()}`] : {};
};

export default localization;
