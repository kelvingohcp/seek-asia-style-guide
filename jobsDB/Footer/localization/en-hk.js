const domain = 'jobsdb.com';
const baseUrl = `https://hk.${domain}`;
const baseUrlWithLanguage = `${baseUrl}/hk/en`;
const wpUrlWithLanguage = `${baseUrl}/en-hk`;
const mbaseUrl = `https://m.${domain}`;
const mUrlWithLanguage = `${mbaseUrl}/en-hk`;

export default {
  'footer.copyright': 'Copyright Â© 1998-{year}, jobsDB.',
  'footer.rightReserved': 'All Rights Reserved.',
  'footer.siteMapLink': `${baseUrlWithLanguage}/sitemap`,
  'footer.siteMapLinkText': 'Site Map',
  'footer.termsLink': `${wpUrlWithLanguage}/pages/terms/terms-conditions`,
  'footer.termsLinkText': 'Terms & Conditions',
  'footer.privacyLink': `${wpUrlWithLanguage}/pages/terms/privacy-policy`,
  'footer.privacyLinkText': 'Privacy Statement',
  'footer.legalLink': `${mUrlWithLanguage}/legal.do`,
  'footer.legalLinkText': 'Legal terms'
};
