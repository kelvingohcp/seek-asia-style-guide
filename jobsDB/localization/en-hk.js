import { FlagHKIcon, FlagIDIcon, FlagSGIcon, FlagTHIcon } from 'seek-asia-style-guide/react';

export default {
  'header.homeTitle': 'Home',
  'header.homeUrl': '/hk',
  'header.mobileLoggedHomeUrl': 'https://m.jobsdb.com/en-hk/jobsForYou.do',
  'header.myJobsDBTitle': 'My jobsDB',
  'header.myJobsDBUrl': '/hk/en/MyJobsDB',
  'header.myJobsDBUrlLoggedIn': '/hk/en/MyJobsDB',
  'header.resourcesTitle': 'Resources',
  'header.resourcesUrl': '/en-hk/career-recruitment-advice/',
  'header.careerInsightsTitle': 'Career Insights',
  'header.careerInsightsUrl': '/hk/en/career-insights',
  'header.employerSiteTitle': 'For Employers',
  'header.employerSiteUrl': '/hk/en/home/employerindex',
  'header.loginTitle': 'Log in',
  'header.loginUrl': '/hk/en/login/jobseekerlogin',
  'header.signupTitle': 'Sign up',
  'header.signupUrl': '/hk/en/jobseeker/registration',
  'header.profileTitle': 'View profile',
  'header.profileUrl': '/hk/en/MyJobsDB/profile',
  'header.invitationTitle': 'Job invitations',
  'header.invitationUrl': '/hk/en/Invitations/InvitationList',
  'header.logoutTitle': 'Log out',
  'header.logoutUrl': 'https://www.thisislogoutlink.test',
  'header.seperatorOr': 'or',
  'header.searchUrl': '/HK/EN/Search/FindJobs?KeyOpt=COMPLEX&JSRV=1&RLRSF=1&JobCat=1&JSSRC=HPSS',
  'menu.jobSeekerHeader': 'JOB SEEKER',
  'menu.jobSeekerProfile': 'PROFILE',
  'menu.settingsHeader': 'SETTINGS',
  'menu.more': 'More',
  'menu.moreHeader': 'MORE',
  'menu.countryAndLanguage': 'Country & Language',
  'menu.backToMenu': 'Back to menu',
  'header.countryList': [ {
    title: 'Hong Kong (English)',
    ItemIcon: FlagHKIcon,
    url: '',
    language: 'en',
    country: 'hk',
    languageDisplay: 'English'
  }, {
    title: 'Indonesia (English)',
    ItemIcon: FlagIDIcon,
    url: '',
    language: 'en',
    country: 'id',
    languageDisplay: 'English'

  }, {
    title: 'Indonesia (Bahasa)',
    ItemIcon: FlagIDIcon,
    url: '',
    language: 'id',
    country: 'id',
    languageDisplay: 'Bahasa'
  }, {
    title: 'Singapore (English)',
    ItemIcon: FlagSGIcon,
    url: '',
    language: 'en',
    country: 'sg',
    languageDisplay: 'English'
  }, {
    title: 'Thailand (English)',
    ItemIcon: FlagTHIcon,
    url: '',
    language: 'en',
    country: 'th',
    languageDisplay: 'English'
  }, {
    title: 'ไทย (ภาษาไทย)',
    ItemIcon: FlagTHIcon,
    url: '',
    language: 'th',
    country: 'th',
    languageDisplay: 'ภาษาไทย'
  } ]
};
