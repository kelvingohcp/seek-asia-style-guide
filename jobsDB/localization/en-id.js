import {
  FlagHKIcon,
  FlagIDIcon,
  FlagSGIcon,
  FlagTHIcon
} from 'seek-asia-style-guide/react';

export default {
  'header.homeTitle': 'Home',
  'header.homeUrl': '/id',
  'header.myJobsDBTitle': 'My jobsDB',
  'header.myJobsDBUrl': '/id/en/MyJobsDB',
  'header.myJobsDBUrlLoggedIn': '/id/en/MyJobsDB',
  'header.resourcesTitle': 'Resources',
  'header.resourcesUrl': '/id-id/career-recruitment-advice/',
  'header.careerInsightsTitle': 'Event',
  'header.careerInsightsUrl': '/id-id/pages/event',
  'header.employerSiteTitle': 'For Employers',
  'header.employerSiteUrl': '/id/en/home/employerindex',
  'header.loginTitle': 'Log in',
  'header.loginUrl': '/id/en/login/jobseekerlogin',
  'header.signupTitle': 'Sign up',
  'header.signupUrl': '/id/en/jobseeker/registration',
  'header.profileTitle': 'View profile',
  'header.profileUrl': '/id/en/MyJobsDB/profile',
  'header.invitationTitle': 'Job invitations',
  'header.invitationUrl': '/id/en/Invitations/InvitationList',
  'header.logoutTitle': 'Log out',
  'header.logoutUrl': 'https://www.thisislogoutlink.test',
  'header.seperatorOr': 'or',
  'header.searchUrl':
    '/ID/EN/Search/FindJobs?KeyOpt=COMPLEX&JSRV=1&RLRSF=1&JobCat=1&JSSRC=HPSS',
  'menu.jobSeekerHeader': 'JOB SEEKER',
  'menu.jobSeekerProfile': 'PROFILE',
  'menu.settingsHeader': 'SETTINGS',
  'menu.more': 'More',
  'menu.moreHeader': 'MORE',
  'menu.countryAndLanguage': 'Country & Language',
  'menu.backToMenu': 'Back to menu',
  'header.countryList': [
    {
      title: 'Hong Kong (English)',
      ItemIcon: FlagHKIcon,
      url: '',
      language: 'en',
      country: 'hk',
      languageDisplay: 'English'
    },
    {
      title: 'Indonesia (English)',
      ItemIcon: FlagIDIcon,
      url: '',
      language: 'en',
      country: 'id',
      languageDisplay: 'English'
    },
    {
      title: 'Indonesia (Bahasa)',
      ItemIcon: FlagIDIcon,
      url: '',
      language: 'id',
      country: 'id',
      languageDisplay: 'Bahasa'
    },
    {
      title: 'Singapore (English)',
      ItemIcon: FlagSGIcon,
      url: '',
      language: 'en',
      country: 'sg',
      languageDisplay: 'English'
    },
    {
      title: 'Thailand (English)',
      ItemIcon: FlagTHIcon,
      url: '',
      language: 'en',
      country: 'th',
      languageDisplay: 'English'
    },
    {
      title: 'ไทย (ภาษาไทย)',
      ItemIcon: FlagTHIcon,
      url: '',
      language: 'th',
      country: 'th',
      languageDisplay: 'ภาษาไทย'
    }
  ]
};
