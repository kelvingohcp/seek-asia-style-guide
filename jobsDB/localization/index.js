import enHkLocalization from './en-hk';
import enSgLocalization from './en-sg';
import enIdLocalization from './en-id';
import idIdLocalization from './id-id';
import enThLocalization from './en-th';
import thThLocalization from './th-th';

const localization = {
  'en-hk': enHkLocalization,
  'en-sg': enSgLocalization,
  'en-id': enIdLocalization,
  'id-id': idIdLocalization,
  'en-th': enThLocalization,
  'th-th': thThLocalization
};

export const getLocalization = ({ language, country }) => {
  return localization[`${language.toLowerCase()}-${country.toLowerCase()}`]
    ? localization[`${language.toLowerCase()}-${country.toLowerCase()}`]
    : {};
};

export default localization;
