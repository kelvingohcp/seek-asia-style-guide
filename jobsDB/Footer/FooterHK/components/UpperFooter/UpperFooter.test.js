import React from 'react';
import { shallow } from 'enzyme';
import UpperFooter from './UpperFooter';
import localization from '../../localization';

describe('UpperFooter:', () => {
  it('should render columns with messages and linking functions', () => {
    const wrapper = shallow(
      <UpperFooter messages={localization['en-hk']} externalLink={() => {}} cfsLink={() => {}} wordpressLink={() => {}} />
    );
    expect(wrapper).toMatchSnapshot();
  });
  it('should render with company profile link', () => {
    const wrapper = shallow(
      <UpperFooter messages={localization['en-hk']} externalLink={() => {}} cfsLink={() => {}} wordpressLink={() => {}} hasCompanyProfile={true} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
