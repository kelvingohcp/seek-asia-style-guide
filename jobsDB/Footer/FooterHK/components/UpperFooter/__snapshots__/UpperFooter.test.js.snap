// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UpperFooter: should render columns with messages and linking functions 1`] = `
<Columns>
  <div
    className="columnGroup"
  >
    <Section
      className="firstCategory"
      header={false}
      pullout={false}
      slim={false}
    >
      <Text
        baseline={true}
        className="title"
        semiStrong={true}
        waving={true}
      >
        About JobsDB
      </Text>
      <PartnerSites
        externalLink={[Function]}
        messages={
          Object {
            "FooterHK.BabaJobLink": "http://www.babajob.com/",
            "FooterHK.BdjobsLink": "http://www.bdjobs.com/",
            "FooterHK.BrighterMondayLink": "http://www.brightermonday.com/",
            "FooterHK.CathoLink": "http://www.catho.com.br/",
            "FooterHK.JobStreetLink": "http://www.jobstreet.com/",
            "FooterHK.JobbermanLink": "http://www.jobberman.com/",
            "FooterHK.JoraLink": "https://hk.jora.com/",
            "FooterHK.ManagerLink": "http://www.manager.com.br/",
            "FooterHK.OCCLink": "https://www.occ.com.mx/",
            "FooterHK.SEEKLink": "http://www.seek.com.au/",
            "FooterHK.SeamanJobsiteLink": "http://www.seamanjobsite.com/",
            "FooterHK.WorkAbroadLink": "http://www.workabroad.ph/",
            "FooterHK.WorkanaLink": "https://www.workana.com/",
            "FooterHK.ZhaopinLink": "http://www.zhaopin.com/",
            "FooterHK.aboutJobsDB": "About jobsDB",
            "FooterHK.aboutJobsDBLink": "/pages/aboutus",
            "FooterHK.advertise": "Advertise with us",
            "FooterHK.advertiseLink": "/cms/employer/product-and-services/",
            "FooterHK.appStore": "jobsDB @ App Store",
            "FooterHK.appStoreLink": "https://itunes.apple.com/hk/app/jobsdb-job-search/id414607432",
            "FooterHK.bestCompanies": "Best Companies",
            "FooterHK.bestCompaniesLink": "/companies",
            "FooterHK.browseJobs": "Browse jobs",
            "FooterHK.browseJobsLink": "/browse",
            "FooterHK.career": "Career @ jobsDB",
            "FooterHK.careerInsights": "Career Insights",
            "FooterHK.careerInsightsTitle": "Career Insights",
            "FooterHK.careerInsightsUrl": "/career-insights",
            "FooterHK.careerLink": "/Search/FindJobs?JSRV=1&AccNums=1&SubAccount=14",
            "FooterHK.cfsLink": "/hk/en",
            "FooterHK.cmsLink": "/en-hk",
            "FooterHK.contactUs": "Contact us",
            "FooterHK.contactUsLink": "/pages/regional-coverage",
            "FooterHK.copyright": "© jobsDB. All Rights Reserved",
            "FooterHK.facebook": "Facebook",
            "FooterHK.facebookLink": "https://www.facebook.com/JobsDB.HongKong",
            "FooterHK.faq": "FAQ",
            "FooterHK.faqLink": "/StaticContentFullContainer/AboutUS/faq/index.htm",
            "FooterHK.googlePlay": "jobsDB @ Google Play",
            "FooterHK.googlePlayLink": "https://play.google.com/store/apps/details?id=com.jobsdb",
            "FooterHK.jobAlerts": "Job Alerts",
            "FooterHK.jobAlertsLink": "/JobAlert/JobAlertCreation",
            "FooterHK.myJobsDBTitle": "My jobsDB",
            "FooterHK.myJobsDBUrl": "/MyJobsDB",
            "FooterHK.partners": "International partners",
            "FooterHK.postJob": "Post a job",
            "FooterHK.postJobLink": "/cms/employer/product-services/job-ads/",
            "FooterHK.privacyLink": "/pages/terms/privacy-policy",
            "FooterHK.privacyLinkText": "Privacy Statement",
            "FooterHK.resumes": "Post resumes",
            "FooterHK.resumesLink": "/Resumes/ResumeIndex",
            "FooterHK.searchCandidates": "Search candidates",
            "FooterHK.searchCandidatesLink": "/cms/employer/product-services/talent-search/",
            "FooterHK.siteMapLink": "/sitemap",
            "FooterHK.siteMapLinkText": "Site Map",
            "FooterHK.termsLink": "/pages/terms/terms-conditions",
            "FooterHK.termsLinkText": "Terms & Conditions",
            "FooterHK.titleAboutJobsDB": "About JobsDB",
            "FooterHK.titleConnect": "Connect",
            "FooterHK.titleEmployer": "Employers",
            "FooterHK.titleJobSeeker": "Job Seekers",
            "FooterHK.twitter": "Twitter",
            "FooterHK.twitterLink": "https://twitter.com/HKJobs",
          }
        }
      />
    </Section>
    <Section
      className="category"
      header={false}
      pullout={false}
      slim={false}
    >
      <Text
        baseline={true}
        className="title"
        semiStrong={true}
        waving={true}
      >
        Job Seekers
      </Text>
    </Section>
  </div>
  <div
    className="columnGroup"
  >
    <Section
      className="category"
      header={false}
      pullout={false}
      slim={false}
    >
      <Text
        baseline={true}
        className="title"
        semiStrong={true}
        waving={true}
      >
        Employers
      </Text>
    </Section>
    <Section
      className="category"
      header={false}
      pullout={false}
      slim={false}
    >
      <Text
        baseline={true}
        className="title"
        semiStrong={true}
        waving={true}
      >
        Connect
      </Text>
    </Section>
  </div>
</Columns>
`;

exports[`UpperFooter: should render with company profile link 1`] = `
<Columns>
  <div
    className="columnGroup"
  >
    <Section
      className="firstCategory"
      header={false}
      pullout={false}
      slim={false}
    >
      <Text
        baseline={true}
        className="title"
        semiStrong={true}
        waving={true}
      >
        About JobsDB
      </Text>
      <PartnerSites
        externalLink={[Function]}
        messages={
          Object {
            "FooterHK.BabaJobLink": "http://www.babajob.com/",
            "FooterHK.BdjobsLink": "http://www.bdjobs.com/",
            "FooterHK.BrighterMondayLink": "http://www.brightermonday.com/",
            "FooterHK.CathoLink": "http://www.catho.com.br/",
            "FooterHK.JobStreetLink": "http://www.jobstreet.com/",
            "FooterHK.JobbermanLink": "http://www.jobberman.com/",
            "FooterHK.JoraLink": "https://hk.jora.com/",
            "FooterHK.ManagerLink": "http://www.manager.com.br/",
            "FooterHK.OCCLink": "https://www.occ.com.mx/",
            "FooterHK.SEEKLink": "http://www.seek.com.au/",
            "FooterHK.SeamanJobsiteLink": "http://www.seamanjobsite.com/",
            "FooterHK.WorkAbroadLink": "http://www.workabroad.ph/",
            "FooterHK.WorkanaLink": "https://www.workana.com/",
            "FooterHK.ZhaopinLink": "http://www.zhaopin.com/",
            "FooterHK.aboutJobsDB": "About jobsDB",
            "FooterHK.aboutJobsDBLink": "/pages/aboutus",
            "FooterHK.advertise": "Advertise with us",
            "FooterHK.advertiseLink": "/cms/employer/product-and-services/",
            "FooterHK.appStore": "jobsDB @ App Store",
            "FooterHK.appStoreLink": "https://itunes.apple.com/hk/app/jobsdb-job-search/id414607432",
            "FooterHK.bestCompanies": "Best Companies",
            "FooterHK.bestCompaniesLink": "/companies",
            "FooterHK.browseJobs": "Browse jobs",
            "FooterHK.browseJobsLink": "/browse",
            "FooterHK.career": "Career @ jobsDB",
            "FooterHK.careerInsights": "Career Insights",
            "FooterHK.careerInsightsTitle": "Career Insights",
            "FooterHK.careerInsightsUrl": "/career-insights",
            "FooterHK.careerLink": "/Search/FindJobs?JSRV=1&AccNums=1&SubAccount=14",
            "FooterHK.cfsLink": "/hk/en",
            "FooterHK.cmsLink": "/en-hk",
            "FooterHK.contactUs": "Contact us",
            "FooterHK.contactUsLink": "/pages/regional-coverage",
            "FooterHK.copyright": "© jobsDB. All Rights Reserved",
            "FooterHK.facebook": "Facebook",
            "FooterHK.facebookLink": "https://www.facebook.com/JobsDB.HongKong",
            "FooterHK.faq": "FAQ",
            "FooterHK.faqLink": "/StaticContentFullContainer/AboutUS/faq/index.htm",
            "FooterHK.googlePlay": "jobsDB @ Google Play",
            "FooterHK.googlePlayLink": "https://play.google.com/store/apps/details?id=com.jobsdb",
            "FooterHK.jobAlerts": "Job Alerts",
            "FooterHK.jobAlertsLink": "/JobAlert/JobAlertCreation",
            "FooterHK.myJobsDBTitle": "My jobsDB",
            "FooterHK.myJobsDBUrl": "/MyJobsDB",
            "FooterHK.partners": "International partners",
            "FooterHK.postJob": "Post a job",
            "FooterHK.postJobLink": "/cms/employer/product-services/job-ads/",
            "FooterHK.privacyLink": "/pages/terms/privacy-policy",
            "FooterHK.privacyLinkText": "Privacy Statement",
            "FooterHK.resumes": "Post resumes",
            "FooterHK.resumesLink": "/Resumes/ResumeIndex",
            "FooterHK.searchCandidates": "Search candidates",
            "FooterHK.searchCandidatesLink": "/cms/employer/product-services/talent-search/",
            "FooterHK.siteMapLink": "/sitemap",
            "FooterHK.siteMapLinkText": "Site Map",
            "FooterHK.termsLink": "/pages/terms/terms-conditions",
            "FooterHK.termsLinkText": "Terms & Conditions",
            "FooterHK.titleAboutJobsDB": "About JobsDB",
            "FooterHK.titleConnect": "Connect",
            "FooterHK.titleEmployer": "Employers",
            "FooterHK.titleJobSeeker": "Job Seekers",
            "FooterHK.twitter": "Twitter",
            "FooterHK.twitterLink": "https://twitter.com/HKJobs",
          }
        }
      />
    </Section>
    <Section
      className="category"
      header={false}
      pullout={false}
      slim={false}
    >
      <Text
        baseline={true}
        className="title"
        semiStrong={true}
        waving={true}
      >
        Job Seekers
      </Text>
    </Section>
  </div>
  <div
    className="columnGroup"
  >
    <Section
      className="category"
      header={false}
      pullout={false}
      slim={false}
    >
      <Text
        baseline={true}
        className="title"
        semiStrong={true}
        waving={true}
      >
        Employers
      </Text>
    </Section>
    <Section
      className="category"
      header={false}
      pullout={false}
      slim={false}
    >
      <Text
        baseline={true}
        className="title"
        semiStrong={true}
        waving={true}
      >
        Connect
      </Text>
    </Section>
  </div>
</Columns>
`;
